packages/toolpad-app/src/toolpad/AppEditor/PageEditor/RenderPanel/RenderOverlay.tsx

```diff
@@ -1365,6 +1365,8 @@ export default function RenderOverlay({ canvasHostRef }: RenderOverlayProps) {
 
         const isResizing = Boolean(draggedEdge) && node.id === draggedNodeId;
 
+        const hasBottomHud = nodeRect.y <= 32;
```

Comment:
`nodeRect` is already passed to `NodeHud`, can't it do the calculation internally?

Reply:
that's correct, i was thinking of another approach first so i didn't consider this - i'll do everything in `NodeHud`

Reply:
done, there's other stuff like this that i can move from `RenderOverlay` into `NodeHud` - i have those noted to do it in the future